docs/notes.md

```diff
@@ -0,0 +1,12 @@
+## Notes
+The player offers a function to add notes to positions in the video. Notes consist of a position and a text, meaning that the user took that note on that position of the video. It's possible to have notes with an empty text that are just used to mark positions. Users can create a with a Note with empty text in the control bar of a player. The note will be shown in the progress bar. Hovering over it will display a tooltip, where users can enter a text or delete that note.
+
+The player does not store the data permanently. Instead, if you want to use this feature, you will have to make an API available for the player that can be used to achieve permanence.  The API may store the data on the server, as a cookie or in the local storage or any other place available. If you do not care about permanence, you may also just implement void operations.
```

Comment:
How this API should be designed? You should give an example.

Link this document to the main documentation

Add Latex Formular execution (but be aware of attacks) ... For JavaScript try to use MathJax ... also available by npm https://www.npmjs.com/package/mathjax

Reply:
The specification of the api (and how to configure) is 2 lines below.

Will add a link.

Latex formula execution - you mean so that users can enter stuff like $a = \sum_{i=0}^{\inf} d_i \cdot p$ into their notes whiel editing and it will be rendered as the latex result when they press return or leave the input field?

Reply:
So you should call a self defined REST API bei the functions load(), add(), ...

Yes right. You should add some math latex with $ and it will be rendered in the paragraph as formula. When you edit it again you see the latex source